lib/migrator.js

```diff
@@ -53,9 +59,22 @@ module.exports = async function(args) {
     for (let i = 0; i < limit; i++) {
       const item = feed.items[i];
       const { date, tags, link } = item;
-      let { title } = item;
-      const content = tomd.turndown(item.content || item.excerpt);
-      const excerpt = tomd.turndown(item.excerpt);
+      let { content, excerpt, title } = item;
+
+      if (excerpt) {
+        if (excerpt.includes('&lt;')) excerpt = unescapeHTML(excerpt);
+        if (content.startsWith(excerpt)) content = content.replace(excerpt + '<a id="more"></a>', '');
```

Comment:
Will `<a id="more">More</a>` always available, or it just the behavior of hexo-generator-feed?

Reply:
`<a id="more"></a>` is added by https://github.com/hexojs/hexo/blob/4931608cc616adfbdfc4e2fd2476bfb3f1ebf8a7/lib/plugins/filter/after_post_render/excerpt.js#L15, I'm just assuming others also use similar tag.
In retrospect, it may not exist, I'll update the code.